elcid/api.py

```diff
@@ -1,5 +1,6 @@
 import datetime
 import re
+import decimal
```

Comment:
We don't need this import ? 
Shouldn't some kind of flake8 check catch that ?

Reply:
👍